modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/BlockSearchRequest.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.rosetta.server.model
+
+case class BlockSearchRequest(
+  isOr: Boolean,
+  isAnd: Boolean,
```

Comment:
it's unsecure. what if `isOr` and `isAnd` will both be true or false?

Reply:
I see that rosetta uses enum `operator: or/and`, not booleans (https://www.rosetta-api.org/docs/models/Operator.html)

Reply:
Changed to use an internal enum.